### DISCUSSION TITLE
nian 社区/论坛/讨论区

### DISCUSSION THREAD

**Discussion by WatsonYao** [WatsonYao/nian_app#1] [version]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LF231987** [comment]
![Uploading 1614919269880.png…]()
很好，加油！
